examples/usecase-passport/keystone.ts

```diff
@@ -0,0 +1,25 @@
+import "dotenv/config";
+import { config } from "@keystone-6/core";
+import { lists } from "./schema";
+import { withAuth, session, createAuthenticationMiddleware } from "./auth";
+import { fixPrismaPath } from "../example-utils";
+
+export default withAuth(
```

Comment:
You shouldn't need to wrap `withAuth` here, I don't see what the keystone auth package is providing here?